versions.tf

```diff
@@ -2,6 +2,6 @@ terraform {
   required_version = ">= 0.13.1"
 
   required_providers {
-    aws = ">= 3.19"
+    aws = ">= 3.44"
```

Comment:
It was added in 3.40 - https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md#3400-may-13-2021

Reply:
@antonbabenko thank you, will update!